docs/docs/configuration/connecting-to-data-stores/lightstep.md

```diff
@@ -8,9 +8,20 @@ Examples of configuring Tracetest with Lightstep can be found in the [`examples`
 
 ## Configuring OpenTelemetry Collector to Send Traces to both Lightstep and Tracetest
 
-In your OpenTelemetry Collector config file, make sure to set the `exporter` to `otlp/tt`, with the `endpoint` pointing to your Tracetest instance on port `21321`. If you are running Tracetest with Docker, the endpoint might look like this `http://tracetest:21321`.
+In your OpenTelemetry Collector config file:
```

Comment:
We will need to do this on our [current Datadog documentation](https://github.com/kubeshop/tracetest/blob/main/docs/docs/configuration/connecting-to-data-stores/datadog.md) too. I can do that on my PR which is adding more examples and docs to Datadog.

Reply:
Makes sense. I will make sure to check where updates are needed once I merge this one.

Reply:
@danielbdias I updated the DD config as well in this PR.